### ISSUE TITLE
about test set

### ISSUE THREAD

**Issue by tngusdldlt** [ustclby/Unsupervised-Domain-Specific-Deblurring#8]
Hi, I had a question while reading your paper. In the section on the data set in your paper, you mentioned the following: "We randomly split the whole dataset into three mutually exclusive subsets: sharp training set (100K images), blurred training set (100K images) and test set (2137 images). For the blurred training set, we use the method in Section 3.6 to blur the images."  But we did not find out exactly what test set you used in this sentence. I would like to know exactly what test set was used to evaluate performance.

**Comment by butcat** [comment]
Hi，I also had the same question. Did you find out the test set?
